From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: "Jan Alexander Steffens (heftig)" <heftig@archlinux.org>
Date: Fri, 3 Mar 2023 18:40:22 +0000
Subject: [PATCH] imagesequencesrc: Properly set default location

Noticed this because the generic_states test kept segfaulting at random.
GLibC 2.37 can crash when NULL is supplied as a format string.
---
 .../gst-plugins-good/gst/multifile/gstimagesequencesrc.c        | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/subprojects/gst-plugins-good/gst/multifile/gstimagesequencesrc.c b/subprojects/gst-plugins-good/gst/multifile/gstimagesequencesrc.c
index 8581e73e3a94..3d59ec74a7c9 100644
--- a/subprojects/gst-plugins-good/gst/multifile/gstimagesequencesrc.c
+++ b/subprojects/gst-plugins-good/gst/multifile/gstimagesequencesrc.c
@@ -356,7 +356,7 @@ gst_image_sequence_src_init (GstImageSequenceSrc * self)
   self->start_index = DEFAULT_START_INDEX;
   self->index = 0;
   self->stop_index = DEFAULT_STOP_INDEX;
-  self->path = NULL;
+  self->path = g_strdup (DEFAULT_LOCATION);
   self->caps = NULL;
   self->n_frames = 0;
   self->fps_n = 30;
